src/uci.cpp

```diff
@@ -286,24 +286,24 @@ void UCI::loop(int argc, char* argv[]) {
           Eval::NNUE::save_eval(filename);
       }
       else if (token == "--help" || token == "help" || token == "--license" || token == "license")
-          sync_cout << "\nStockfish is a powerful chess engine and free software licensed under the GNU GPLv3."
-                       "\nStockfish is normally used with a separate graphical user interface (GUI)."
-                       "\nStockfish implements the universal chess interface (UCI) to exchange information."
-                       "\nFor further information see https://github.com/official-stockfish/Stockfish#readme"
-                       "\nor the corresponding README.md and Copying.txt files distributed with this program.\n" << sync_endl;
+          sync_cout << "\nStockfish is a powerful chess engine providing NNUE evaluation."
+                       "\nIt is released as free software licensed under the GNU GPLv3 License."
+                       "\nStockfish is normally used with a graphical user interface (GUI) and implements"
+                       "\nthe Universal Chess Interface (UCI) protocol to communicate with a GUI program."
```

Comment:
`to communicate with a GUI program.` is there some better formulation, i.e. not restricting this to GUIs ? I don't think that `to exchange information` was so bad, maybe there are alternatives.